chaosaws/rds/actions.py

```diff
@@ -191,6 +192,46 @@ def delete_db_cluster_endpoint(db_cluster_identifier: str,
             db_cluster_identifier, e.response['Error']['Message']))
 
 
+def start_db_instance(
+    db_instance_identifier: str,
+    configuration: Configuration = None,
+    secrets: Secrets = None
+) -> AWSResponse:
+    """
+    Start a RDS DB instance
+
+    - db_instance_identifier: the identifier of the RDS instance to start
+    """
+    client = aws_client("rds", configuration, secrets)
+
+    params = dict(
+        DBInstanceIdentifier=db_instance_identifier)
+
+    try:
+        call = client.start_db_instance(**params)
+        timeout = 5
+        timeout_max = 600
+
+        while (
+            client.describe_db_instances(
+                DBInstanceIdentifier=db_instance_identifier
+            )['DBInstances'][0]['DBInstanceStatus'] != 'available'
+        ):
+            if timeout <= timeout_max:
```

Comment:
I usually don't do it this way:

instead:

before the while loop:

`until = time.time() + timeout`

then during the loop:

```
if time.time() >= until:
   raise FailedActivity(...)
```